prog/vm/nexus.rb

```diff
@@ -99,7 +99,8 @@ def self.assemble(public_key, project_id, name: nil, size: "standard-2",
           "storage_volumes" => storage_volumes.map { |v| v.transform_keys(&:to_s) },
           "swap_size_bytes" => swap_size_bytes,
           "distinct_storage_devices" => distinct_storage_devices,
-          "force_host_id" => force_host_id
+          "force_host_id" => force_host_id,
+          "gpu_enabled" => gpu_enabled
```

Comment:
Where do we use this in the Vm:Nexus?

Reply:
It'll be used in a future PR. We are stitching it from both ends with @bsatzger